### DISCUSSION TITLE
Automated Deployment

### DISCUSSION THREAD

**Discussion by bugdiver** [BugDiver/gauge-ts#172] [General]
@zabil I have cloned all the labels from another plugins repo to this one. Now any PR which is meant to trigger a deployment can be tagged with `ReleaseCandidate` and when merged will trigger a deployment. 
Currently this is not functioning properly, because by default gauge-bot create deployment for `master` branch only, but this repo uses `main` and default branch.
There are two way to fix this:
* [x] Update gauge-bot to create deployment against the default branch
* [ ] Change the default brach from `main` to `master`

What do you suggest?

**Comment by zabil** [comment]
I think it’ll be good to have the branch name as main. So update gauge bot?

**Reply by bugdiver** [reply]
Cool, here the PR for the same, Not sure how netlify deployments are done

https://github.com/getgauge/gauge-bot/pull/34

**Reply by zabil** [reply]
Your netlify access was on an old tw email. Sent you a new invite. You should get access to Gauge team.

**Reply by bugdiver** [reply]
Got it, thanks, I have merged the PR. Let's try to create a new PR in `gauge-ts` and label it as RC, hopefully the deployment should trigger when it's merged

**Reply by zabil** [reply]
Ok let me raise a new PR with some doc updates
